packages/typespec-go/src/tcgcadapter/types.ts

```diff
@@ -304,6 +306,7 @@ export class typeAdapter {
         this.types.set(etagKey, etag);
         return etag;
       }
+      case 'float': // C# and Java define float as 32 bits so we're following suit
```

Comment:
This should be in the TypeSpec specification.  Is there a bug to spec out the size of a float?

Reply:
Unfortunately, the size is absent, see the [docs](https://typespec.io/docs/language-basics/built-in-types#numeric-types). It could be the case that this is language dependent (if it is, the TS docs should call this out).

Reply:
FYI @bterlson